creator-node/src/services/sync/processSync.js

```diff
@@ -353,7 +352,8 @@ async function processSync (serviceRegistry, walletPublicKeys, creatorNodeEndpoi
       let redisKey = redis.getNodeSyncRedisKey(wallet)
       await redisLock.removeLock(redisKey)
     }
-    logger.info(redisKey, `DURATION SYNC ${Date.now() - start}`)
+
+    logger.error(redisKey, `Sync complete for wallets: ${walletPublicKeys.join(',')}. Status: ${errorObj ? 'Error: ' + errorObj.message : 'Success'}. Duration sync: ${Date.now() - start}. From endpoint ${creatorNodeEndpoint}.`)
   }
 
```

Comment:
ah i just remembered - pretty sure i introduced a bug in the write quorum PR that will break this logic. specifically that `errorObj` isn't always an object with a `message` field. look at line 41:
```
const errorObj = `Cannot change state of wallet ${wallet}. Node sync currently in progress.`
```
i think this should probably be:
```
errorObj = { message: `Cannot change state of wallet ${wallet}. Node sync currently in progress.` }
```
right?

Reply:
can we just create an error like let errorObj = new Error(`Cannot change...`)

Reply:
ah yeah thats the correct way

Reply:
it would be so silly to have our logging break + reject the sync bc of this lol